main.go

```diff
@@ -35,6 +35,13 @@ to quickly create a Cobra application.`,
 	rootCmd.PersistentFlags().StringVar(&config.CloudAwsSecretId, "cloud-aws-secret-id", "", "Use AWS Secrets Manager for Config. If set it pull the environment variables from aws secrets manager.")
 	rootCmd.PersistentFlags().StringArrayVar(&config.AppAwsSecretIds, "app-aws-secret-ids", []string{}, "Ex. 1234.dkr.ecr.us-west-2.amazonaws.com")
 
+	//GCP KMS Variables Here
+	rootCmd.PersistentFlags().StringVar(&config.GCPEncryptedSecretsFile, "cloud-gcp-kms-encrypted-file", "", "the path of the encrypted kms file")
```

Comment:
These fields don't follow what I wanted. Reread the args I want:

https://github.com/opszero/deploytag/issues/8